packages/@okta/vuepress-site/docs/guides/oin-oidc-guide/protocol-level-requirements/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Protocol-level requirements
+---
+
+### Code flow
+
+For OIDC applications destined for the OIN, you are building a "web" application with a dedicated server-side back end that is capable of securely storing a Client Secret and exchanging information with an authorization server through trusted back-channel connections.
+
+The exchange of the short-lived authorization code for a token requires a Client Secret. This is a value that only your Client and the authorization server know and must be protected from malicious parties.
```

Comment:
that only your Client -> that only your client application.
Should we have a caveat somewhere in the beginning that we can interchangeably refer to the OIN application being submitted as the OIDC client application (ie use the OIDC lingo or terminology)